packages/web/src/scss/theme/_form-fields.scss

```diff
@@ -0,0 +1,41 @@
+@use '@tokens' as tokens;
+
+$spacing: tokens.$space-300;
+
+$input-typography: tokens.$body-medium-text-regular;
+$input-color-default: tokens.$text-primary-default;
+$input-color-disabled: tokens.$text-primary-disabled;
+$input-color-unchecked: tokens.$action-unselected-default;
+$input-color-checked: tokens.$action-selected-default;
+$input-border-width: tokens.$border-width-100;
+$input-border-style: tokens.$border-style-100;
+$input-border-color-default: tokens.$border-primary-default;
+$input-border-color-hover: tokens.$border-primary-hover;
+$input-border-color-focus: tokens.$focus-default;
+$input-border-color-disabled: tokens.$border-primary-disabled;
+$input-border-color-success: tokens.$emotion-success-default;
+$input-border-color-warning: tokens.$emotion-warning-default;
+$input-border-color-error: tokens.$emotion-danger-default;
+$input-background: tokens.$background-basic;
+$input-focus-shadow: tokens.$focus;
+$input-placeholder-color-default: tokens.$text-secondary-default;
+$input-placeholder-color-disabled: tokens.$text-secondary-disabled;
+$input-margin: 3px;
```

Comment:
It seems it only affects checkboxes and radios. Do you think you could distinguish somehow between what's common to all fields, what's for text fields, and what's just for checkboxes and radios?